Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -735,7 +735,7 @@ void OnListItemClicked(object sender, ItemClickEventArgs e)
 				var templatedItems = TemplatedItemsView.TemplatedItems;
 				var selectedItemIndex = templatedItems.GetGlobalIndexOfItem(e.ClickedItem);
 
-				OnListItemClicked(selectedItemIndex, e.ClickedItem as Cell);
```

Comment:
ClickedItem contains the BindingContext not the Cell.  It's only a cell if the BindingContext is also a Cell which isn't a case where we would still want to pass it in as the cell argument. *NotifyRowTapped* needs to just locate the Cell itself where the event fires. Which is also broken on UWP but this change makes it less broken.

This scenario was fairly edge case anyway because since ClickedItem is the BindingContext it will almost never be a Cell unless it's binding to itself or the BindingContext for some reason implements Cell (which it normally won't). So in almost all cases this cast will just be null and in the cases where it's not null then it's passing the wrong Cell into *NotifyRowTapped*